docker/local/go-ethereum.Dockerfile

```diff
@@ -4,10 +4,12 @@ RUN apk update ; apk add --no-cache --allow-untrusted ca-certificates curl bash
 RUN apk add --no-cache --upgrade grep
 WORKDIR /root
 
+ARG GENESIS_FILE_PATH=start-up-files/go-ethereum/genesis.json
```

Comment:
This looks good to me, although I have always found it easier to use a volume, but if this works then I am happy to implement it.

Reply:
I have just added argument for specifying the file path for `genesis.json` file which is used in the lines below.
Since [USE_GENESIS](https://github.com/vulcanize/stack-orchestrator/blob/main/docker/local/docker-compose-go-ethereum.yml#L28) option is set to true, it is using the custom [genesis.json](https://github.com/vulcanize/stack-orchestrator/blob/main/start-up-files/go-ethereum/genesis.json) file.
So with `GENESIS_FILE_PATH` argument different file can be specified. For watcher demo, the blocks need to be mined in intervals and so `auto-genesis.json` file is added with `clique.period` set to 3.

Not sure if by using volume the required genesis file can be mounted.